templates/cassiopeia/scss/tools/variables/_variables.scss

```
@@ -148,7 +148,7 @@ $cassiopeia-grid-gutter:              $grid-gutter-width;
 
 // Components
 $fa-font-path:                        "../../../media/vendor/fontawesome-free/webfonts";
-
+$fa-fw-width:                         20em * .0625;
```

```suggestion
$fa-fw-width:                         20em * .0625;

```

keep the blank line between sections